server/api/api.go

```diff
@@ -1322,27 +1323,29 @@ func (a *API) errorResponseWithCode(w http.ResponseWriter, statusCode int, error
 		data = []byte("{}")
 	}
 	w.WriteHeader(statusCode)
-	w.Write(data)
+	_, _ = w.Write(data)
 }
 
 func (a *API) noContainerErrorResponse(w http.ResponseWriter, sourceError error) {
-	a.errorResponseWithCode(w, http.StatusBadRequest, ERROR_NO_WORKSPACE_CODE, ERROR_NO_WORKSPACE_MESSAGE, sourceError)
+	a.errorResponseWithCode(w, http.StatusBadRequest, ErrorNoWorkspaceCode, ErrorNoWorkspaceMessage, sourceError)
 }
 
-func jsonStringResponse(w http.ResponseWriter, code int, message string) {
+func jsonStringResponse(w http.ResponseWriter, code int, message string) { //nolint:unparam
```

Comment:
why is needed the `unparam` here? looks like all the 3 parameters are used.

Reply:
`unparam` also checks if any params are always called with the same value.  For example, `jsonStringResponse` is called using code=200 in all cases currently, but the method should remain generic IMO.

Reply:
That sounds right, other option is to remove the code parameter and add it when is needed, instead of giving the function a flexibility that we are not using now. Anyway... It is ok to have the `//nolint` for now. I guess sooner than later we are going to need the code there.